py3status/modules/speedtest.py

```diff
@@ -265,14 +255,6 @@ def _start(self):
         self.start_time = None
         self.cached_until = self.py3.CACHE_FOREVER
 
-        # cast number
-        self.speedtest_data.update(
-            {
-                k: self._cast_number(self.speedtest_data[k])
```

Comment:
Is this correct? I was talking about casting `elapsed_time` alone.  Just a heads up.

Reply:
My try was good with default format, but i have not try other placeholders after this commit. Maybe only placeholder update in Meta class are good.